graph-tutorial/src/webparts/graphTutorial/GraphTutorialWebPart.ts

```diff
@@ -1,17 +1,17 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
Be sure to include this copyright/license comment block in all your source files.